device/build/build.cmd

```diff
@@ -35,15 +35,15 @@ rem // default build options
 set build-clean=0
 set build-config=Release
 set build-platform="Any CPU"
-set wip-use-c2d-amqp-methods=0
```

Comment:
Not sure why we are removing all the use-c2d-amqp-methods calls.

Reply:
methods over amqp is released.  all corresponding compile flags are removed and I am just cleaning them up.